### PR TITLE
added a new option for timelines

### LINKED ISSUE #589: Add “Title - account icon - username - domain name” to Timelines → header display
Presently, the options under Timelines → header display include “Title (domain.name)”, “Title (username@domain.name)”, “Title | account icon (domain.name)”, “Title | account icon”, and “Title”. I think it would be nice if we combined the two most informative options to create a new option, “Title | account icon (username@domain.name)”, making as easy as possible to tell at a glance which column is for which account.

### PR DESCRIPTION
**Pull request by HamzaFarooqArif**
added a new option for timelines for 'Title - account icon - username - domain name'
fixes #589

### CODE REVIEW

File: src/app/components/stream/stream.component.html [hunk 1]
```diff
@@ -7,11 +7,11 @@
...
+                <img *ngIf="timelineHeader === 3 || timelineHeader === 4 || timelineHeader === 6" class="stream-column__stream-selector--avatar" src="{{avatar}}" />
                 <fa-icon class="stream-column__stream-selector--icon" [icon]="columnFaIcon"></fa-icon>
                 <span class="stream-column__stream-selector--text">
-                    <h1 class="stream-column__stream-selector--title" [class.stream-column__stream-selector--title--only]="timelineHeader === 4 || timelineHeader === 5">{{ streamElement.name.toUpperCase() }}</h1>
-                    <span class="stream-column__stream-selector--subtitle" *ngIf="streamElement.instance && timelineHeader !== 4 && timelineHeader !== 5"><span *ngIf="timelineHeader === 2">{{account.username}}@</span>{{ streamElement.instance.toLowerCase() }}</span>
+                    <h1 class="stream-column__stream-selector--title" [class.stream-column__stream-selector--title--only]="timelineHeader === 4 || timelineHeader === 5 || timelineHeader === 6">{{ streamElement.name.toUpperCase() }}</h1>
```
Comment (NicolasConstant) [inline src/app/components/stream/stream.component.html:13]: this break the spacing between the title and the username, we should keep the older code on this one.

**Comment by NicolasConstant** [comment]
Thanks for your contribution! I'll try to merge/release this in the following days. 🙂

**Comment by NicolasConstant** [comment]
(also, if you share me a fediverse handle, I'll credit you with the Sengi's mastodon account when publishing the update)

**Comment by HamzaFarooqArif** [comment]
> (also, if you share me a fediverse handle, I'll credit you with the Sengi's mastodon account when publishing the update)

@hamzafarooqarif@mastodon.social

File: src/app/components/stream/stream.component.html (hunk 1 above)
Comment (HamzaFarooqArif) [reply src/app/components/stream/stream.component.html]: Changes done. please review again

**Pull request merged**